packages/k6-generator/src/unique/user.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This file before was too large. Almost 15MB. If we need one with that many records, can we dynamically make them? Why do they need to be in a static file.

Reply:
Dynamically creating the information was an issue I ran into due to the nature of k6 and imports, but I can revisit this. 

Now that I've nailed the deleting/logging out during the actual test though, the size of this file can be drastically reduced with no impact on the test - I just have to rerun the model generator for a smaller output